Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM debian
-RUN apt-get update
-RUN apt-get install -y ca-certificates
-RUN apt-get install -y curl
-RUN apt-get install -y iputils-ping
-RUN apt-get install -y jq
+RUN set -x && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+      ca-certificates curl iputils-ping jq && \
```

Comment:
I think we can remove JQ since we don't use it in the integration test anymore, right? what about the other packages, are they still needed?

Reply:
Let's leave them, this is a dev-only docker - these will be useful for debugging.